spec/lib/rmt/cli/import_spec.rb

```diff
@@ -18,13 +19,19 @@
     end
 
     context 'with existing lockfile' do
-      before { allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError) }
+      before do
+        allow(RMT::Lockfile).to receive(:create_file).and_raise(RMT::Lockfile::ExecutionLockedError)
+      end
 
       it 'handles lockfile exception' do
         FakeFS.with_fresh do
           FileUtils.mkdir_p path
 
-          expect { command }.to output(/Process is locked/).to_stdout.and output('').to_stderr
+          expect(described_class).to receive(:exit)
+          expect(File).to receive(:read).with(RMT::Lockfile::LOCKFILE_LOCATION).and_return(pid)
```

Comment:
Same as with the other spec: If possible, create a real lockfile and test with that!

Reply:
changed to stubbing the `flock` method